test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -1010,4 +1012,83 @@ public void reconnectDisconnectedClient() throws Exception {
         shutdownCorfuServer(corfuServer_2);
         shutdownCorfuServer(corfuServer_3);
     }
+
+    /**
+     * Tests a force remove after 2 nodes have been shutdown and the remaining one node
+     * has sealed itself.
+     * Setup: 3 node cluster.
+     * 2 nodes PORT_1 and PORT_2 are shutdown.
+     * A runtime attempts to write data into the cluster with a registered systemDownHandler.
+     * Once the systemDownHandler is invoked, we can be assured that the 2 nodes are shutdown.
+     * We verify that the remaining one node PORT_0 has sealed itself by sending a
+     * NodeStatusRequest. This is responded by a WrongEpochException.
+     * Finally we attempt to force remove the 2 shutdown nodes.
+     */
+    @Test
+    public void forceRemoveAfterSeal() throws Exception {
+
+        // Set up cluster of 3 nodes.
+        final int PORT_0 = 9000;
+        final int PORT_1 = 9001;
+        final int PORT_2 = 9002;
+        Process corfuServer_1 = runPersistentServer(corfuSingleNodeHost, PORT_0, false);
+        Process corfuServer_2 = runPersistentServer(corfuSingleNodeHost, PORT_1, false);
+        Process corfuServer_3 = runPersistentServer(corfuSingleNodeHost, PORT_2, false);
+        List<Process> corfuServers = Arrays.asList(corfuServer_1, corfuServer_2, corfuServer_3);
+        final Layout layout = getLayout(3);
+        final int retries = 3;
+        Sleep.SECONDS.sleepUninterruptibly(1);
+        BootstrapUtil.bootstrap(layout, retries, PARAMETERS.TIMEOUT_SHORT);
+
+        final int systemDownHandlerLimit = 10;
+        runtime = CorfuRuntime.fromParameters(CorfuRuntimeParameters.builder()
```

Comment:
Just for my understanding.
How does runtime know here about corfuServers? I don't see it being passed here.

Reply:
It doesn't need to know about all the servers. I've passed one of the nodes as a connection endpoint `.layoutServer(NodeLocator.parseString(DEFAULT_ENDPOINT))`. Once the runtime connects to this node, it can fetch the layout and get info about all the servers in the cluster.